### PR TITLE
PonyORM tutorial

### LINKED ISSUE #6043: [Docs] PonyORM Tutorial
Background: https://airtable.com/tblD3oZPLJgGhCmch/viw1DKmbKhg2MIECH/recxyyOwodTI2twsK

Description: We now have a sample getting started guide for using this ORM.

CockroachDB will support the PonyORM in 20.1. Currently, there is no getting started tutorial for this ORM. 

Team: Andrew Woods, Rafi Shamim, Eric Harmeling

Github Tracking Issue: 

[UPDATE 2/4]: Code sample delivered.

### PR DESCRIPTION
**Pull request by ericharmeling**
Fixes #6043 .

- Added page for PonyORM tutorial
- Added tab for PonyORM in other Python doc pages
- Added PonyORM to list of Python ORMs

I tested the code in insecure and secure mode. The code works just fine. Pony implemented a retry decorator that they use for all transactions, which lightens the load on us and on users (thanks Pony!).

**Note**
I won't merge this until we officially launch support.

### CODE REVIEW

**Comment by awoods187** [comment]
Let's add this to 19.2 as well!

**Comment by ericharmeling** [comment]
> Let's add this to 19.2 as well!

@awoods187 
Done!

**Comment by ericharmeling** [comment]
@apantel @rafiss Ping on review :) Just one of you needs to look at this really.

File: v19.2/build-a-python-app-with-cockroachdb-pony.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+{% include {{page.version.version}}/app/insecure/create-maxroach-user-and-bank-database.md %}
+
+## Step 3. Run the Python code
+
+</section>
+
+The code below uses PonyORM to map Python objects and methods to SQL operations. When you run the code as a script, it performs the following operations:
+
+1. Reads existing account IDs from the bank database.
```
Comment (Amruta-Ranade) [inline v19.2/build-a-python-app-with-cockroachdb-pony.md]: nit: bank > `bank`

File: v19.2/build-a-python-app-with-cockroachdb-pony.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+</section>
+
+Run the code:
+
+{% include copy-clipboard.html %}
+~~~ shell
+$ python3 pony-basic-sample.py
+~~~
+
+To verify that the table and rows were created successfully, open a new terminal, and start a new session with the built-in SQL client:
```
Comment (Amruta-Ranade) [inline v19.2/build-a-python-app-with-cockroachdb-pony.md]: nit: remove the comma after "terminal"

File: v19.2/build-a-python-app-with-cockroachdb-pony.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
+---
+title: Build a Python App with CockroachDB and PonyORM
+summary: Learn how to use CockroachDB from a simple Python application with PonyORM.
+toc: true
+twitter: false
+---
+
+<div class="filters filters-big clearfix">
+    <a href="build-a-python-app-with-cockroachdb.html"><button style="width: 20%" class="filter-button">Use <strong>psycopg2</strong></button></a>
+    <a href="build-a-python-app-with-cockroachdb-sqlalchemy.html"><button style="width: 20%" class="filter-button">Use <strong>SQLAlchemy</strong></button></a>
```
Comment (Amruta-Ranade) [inline v19.2/build-a-python-app-with-cockroachdb-pony.md]: nit: The `Use SQLAlchemy` tab looks horizontally off-center in the online preview.

File: v19.2/build-a-python-app-with-cockroachdb-pony.md [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+{{site.data.alerts.end}}
+
+
+## Step 1. Install PonyORM
+
+To install PonyORM run the following command:
+
+{% include copy-clipboard.html %}
+~~~ shell
+$ pip install pony
```
Comment (Amruta-Ranade) [inline v19.2/build-a-python-app-with-cockroachdb-pony.md]: nit: Same issue as with Django -- might want to remind users to use `pip3` if `pip` doesn't work for them.

File: v19.2/build-a-python-app-with-cockroachdb-pony.md [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+
+<section class="filter-content" markdown="1" data-scope="secure">
+
+## Step 2. Create the `maxroach` user and `bank` database
+
+{% include {{page.version.version}}/app/create-maxroach-user-and-bank-database.md %}
+
+## Step 3. Generate a certificate for the `maxroach` user
+
+Create a certificate and key for the `maxroach` user by running the following command. The code samples will run as this user.
```
Comment (Amruta-Ranade) [inline v19.2/build-a-python-app-with-cockroachdb-pony.md]: Create a certificate > Create a client certificate

File: v19.2/build-a-python-app-with-cockroachdb-pony.md [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+The code below uses PonyORM to map Python objects and methods to SQL operations. When you run the code as a script, it performs the following operations:
+
+1. Reads existing account IDs from the bank database.
+2. Creates additional accounts with randomly generated IDs, and then adds a bit of money to each new account.
+3. Chooses two accounts at random and takes half of the money from the first account and deposits it into the second.
+
+<section class="filter-content" markdown="1" data-scope="secure">
+
+Copy the code below to a file or
+<a href="https://raw.githubusercontent.com/cockroachdb/docs/master/_includes/{{page.version.version}}/app/pony-basic-sample.py">download it directly</a>.
```
Comment (Amruta-Ranade) [inline v19.2/build-a-python-app-with-cockroachdb-pony.md]: This code looks weird here but renders okay ¯\_(ツ)_/¯

File: v19.2/build-a-python-app-with-cockroachdb-pony.md [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+> SELECT COUNT(*) FROM accounts;
+~~~
+
+~~~
+ count
+-------
+   100
+(1 row)
+~~~
+
```
Comment (Amruta-Ranade) [inline v19.2/build-a-python-app-with-cockroachdb-pony.md]: The code works perfectly!

File: v20.1/build-a-python-app-with-cockroachdb-pony.md [hunk 8]
```diff
@@ -0,0 +1,140 @@
+---
+title: Build a Python App with CockroachDB and PonyORM
```
Comment (Amruta-Ranade) [inline v20.1/build-a-python-app-with-cockroachdb-pony.md]: All the comments for the 19.2 doc apply to 20.1

**Pull request merged**